### PR TITLE
NF: NonNull note

### PR DESCRIPTION
**Pull request by Arthur-Milchior**

### CODE REVIEW

File: AnkiDroid/src/androidTest/java/com/ichi2/anki/tests/ContentProviderTest.java [hunk 1]
```diff
@@ -25,6 +25,8 @@
 import android.content.ContentValues;
 import android.database.Cursor;
 import android.net.Uri;
+
+import androidx.annotation.NonNull;
```
Comment (david-allison-1) [inline AnkiDroid/src/androidTest/java/com/ichi2/anki/tests/ContentProviderTest.java:29]: not needed

File: AnkiDroid/src/main/java/com/ichi2/anki/CardTemplatePreviewer.java [hunk 2]
```diff
@@ -31,6 +31,7 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import androidx.annotation.NonNull;
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/anki/CardTemplatePreviewer.java:34]: not needed

File: AnkiDroid/src/test/java/com/ichi2/anki/AbstractFlashcardViewerTest.java [hunk 3]
```diff
@@ -11,6 +11,7 @@
 import org.robolectric.Robolectric;
 import org.robolectric.android.controller.ActivityController;
 
+import androidx.annotation.NonNull;
```
Comment (david-allison-1) [inline AnkiDroid/src/test/java/com/ichi2/anki/AbstractFlashcardViewerTest.java:14]: not needed

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
nits, but main change is LGTM

**Comment by Arthur-Milchior** [comment]
Done

**Review by david-allison-1 — APPROVED** [review]
LGTM

**Pull request merged**